### PR TITLE
Add test to make sure window.context.data is the same data object passed into 3P extension code.

### LINKED ISSUE #10628: Add test to make sure window.context.data is the same data object passed into 3P extension code
This is to follow up a production bug we encountered: #10354

/cc @alanorozco

### PR DESCRIPTION
**Pull request by lannka**
closes #10628

### CODE REVIEW

File: ads/_ping_.js [hunk 1]
```diff
@@ -18,10 +18,13 @@ import {validateData} from '../3p/3p';
 import {dev} from '../src/log';
 
 /**
+ * A fake ad network integration that is mainly used for testing
+ * and demo purpose.
  * @param {!Window} global
  * @param {!Object} data
  */
 export function _ping_(global, data) {
+  global.dataAsParam = data; // for testing only. see #10628
```
Comment (alanorozco) [inline ads/_ping_.js]: Can you rename to `networkIntegrationDataParamForTesting`?

I know it's a mouthful, but this particular thing is really weird and I don't want future me to get confused :)

File: ads/_ping_.js [hunk 2]
```diff
@@ -18,10 +18,13 @@ import {validateData} from '../3p/3p';
 import {dev} from '../src/log';
 
 /**
+ * A fake ad network integration that is mainly used for testing
+ * and demo purpose.
```
Comment (alanorozco) [inline ads/_ping_.js]: nit: "demo purposes" in plural

Also add "This implementation gets stripped out in compiled production code."

Comment (lannka) [inline ads/_ping_.js]: done

File: ads/_ping_.js (hunk 1 above)
Comment (lannka) [inline ads/_ping_.js]: done

**Pull request merged**